apigw-lambda-translate/README.md

```diff
@@ -0,0 +1,95 @@
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd apigw-lambda-translate
+    ```
+3. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy --guided
+    ```
+4. During the prompts:
+    * Enter a stack name
+    * Enter the desired AWS Region
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+5. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs that were created.
+
+## How it works
+
+This pattern deploys an Amazon API Gateway REST API and a default route integrated with an AWS Lambda function written in Python. The lambda function calls AWS Translate service using the text and language given in the request body and returns the translated text or error response if the language code is not supported.
+
+## Testing
+
+Once the application is deployed, either use a curl or call the endpoint from Postman.
+
+Example POST Request to translate text to Spanish:
+    curl -X POST "https://{api-gateway-endpoint}/prod"  -H 'Content-Type: application/json' -d '{"OriginalText":"Hello, my name is AWS","TranslateToLanguage":"es"}'
+
+Response:
+```
+  "Hola, mi nombre es AWS"
+```
+
+Example POST Request to translate text to Afrikaans:
+    curl -X POST "https://{api-gateway-endpoint}/prod"  -H 'Content-Type: application/json' -d '{"OriginalText":"Hello, my name is AWS","TranslateToLanguage":"af"}'
+
+Response:
+```
+  "Hallo, my naam is AWS"
+```
+
+Example POST Request with unsupported language:
+    curl -X POST "https://{api-gateway-endpoint}/prod"  -H 'Content-Type: application/json' -d '{"OriginalText":"Hello, my name is AWS","TranslateToLanguage":"ef"}'
+
+Response:
+```
+  "An error occurred (UnsupportedLanguagePairException) when calling the TranslateText operation: Unsupported language pair: en to ef. Target language 'ef' is not supported"
+```
+
+## Documentation
+- [Working with HTTP APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api.html)
+- [Working with AWS Lambda proxy integrations for HTTP APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html)
+- [AWS Lambda - the Basics](https://docs.aws.amazon.com/whitepapers/latest/serverless-architectures-lambda/aws-lambdathe-basics.html)
+- [Lambda Function Handler](https://docs.aws.amazon.com/whitepapers/latest/serverless-architectures-lambda/the-handler.html)
+- [Function Environment Variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
+
+## Cleanup
+
+1. Delete the stack
+    ```bash
+    aws cloudformation delete-stack --stack-name STACK_NAME
```

Comment:
you can use sam delete nowadays to remove a stack, please update the readme

Reply:
updated with sam delete